defaults/k8s_local_kind_config.yaml

```diff
@@ -27,6 +27,7 @@ k8s_use_chaos_mesh: true
 # NOTE: if we do not specify 'k8s_n_loader_pods_per_cluster' then value of the 'n_loaders' is used
 n_loaders: 1
 n_monitor_nodes: 1
+k8s_n_monitor_nodes: 0
```

Comment:
shouldn't we have at least one test case with this enabled ? 
or we plan to enable it only for EKS based job ?